### PR TITLE
Ref normalize configuration

### PR DESCRIPTION
**Pull request by keichwa**
close https://github.com/SUSE/spacewalk/issues/10590

### CODE REVIEW

**Comment by keichwa** [comment]
- I move some parts to the Client Config.  Or would Admin a better place?
- Is the rest suitable for the Reference?
- Thus far, I did not convert the lists to formal tables.  I probably should do this.

Thanks for help and advice!

**Comment by Loquacity** [comment]
> * I move some parts to the Client Config.  Or would Admin a better place?

No, I think Client Config is fine. 

> * Is the rest suitable for the Reference?

Seems OK at a glance. I think you need to do a couple of "For more on configuration management, see xref:client-config ... "

> * Thus far, I did not convert the lists to formal tables.  I probably should do this.

Yes, for at least some of them. Use your best judgement :)

**Comment by Loquacity** [comment]
@keichwa Status on this?

**Comment by keichwa** [comment]
On 2020-04-27 05:12, Lana Brindley wrote:
> @keichwa Status on this?
Partially done.  Still on hold.
-- 
Karl Eichwalder

File: modules/reference/pages/configuration/config-overview.adoc [hunk 1]
```diff
@@ -10,15 +10,28 @@ image::configuration_overview.png[scaledwidth=80%]
 
 Configuration Summary::
 Provides quick information about your configuration files.
-Click the blue text to the right to display relevant systems, channel details, or configuration files.
+Click the blue text to the right to display:
++
+* Systems with Managed Configuration Files
```
Comment (Loquacity) [inline modules/reference/pages/configuration/config-overview.adoc:15]:
```suggestion
* Systems with managed configuration files
```

File: modules/reference/pages/configuration/config-overview.adoc [hunk 2]
```diff
@@ -10,15 +10,28 @@ image::configuration_overview.png[scaledwidth=80%]
...
+* Configuration Channels
```
Comment (Loquacity) [inline modules/reference/pages/configuration/config-overview.adoc:16]:
```suggestion
* Configuration channels
```

File: modules/reference/pages/configuration/config-overview.adoc [hunk 3]
```diff
@@ -10,15 +10,28 @@ image::configuration_overview.png[scaledwidth=80%]
...
+* Centrally-managed Configuration Files
```
Comment (Loquacity) [inline modules/reference/pages/configuration/config-overview.adoc:17]:
```suggestion
* Centrally-managed configuration files
```

File: modules/reference/pages/configuration/config-overview.adoc [hunk 4]
```diff
@@ -10,15 +10,28 @@ image::configuration_overview.png[scaledwidth=80%]
...
+* Locally-managed Configuration Files
```
Comment (Loquacity) [inline modules/reference/pages/configuration/config-overview.adoc:18]:
```suggestion
* Locally-managed configuration files
```

File: modules/reference/pages/configuration/config-overview.adoc [hunk 5]
```diff
@@ -10,15 +10,28 @@ image::configuration_overview.png[scaledwidth=80%]
...
+
 
 Configuration Actions::
-[guimenu]``Configuration Actions`` offers direct access to the most common configuration management tasks.
-Deploy, compare, or create files on your systems.
+[guimenu]``Configuration Actions`` offers direct access to the most common configuration management tasks:
++
+* View Systems with Managed Configuration Files
```
Comment (Loquacity) [inline modules/reference/pages/configuration/config-overview.adoc:24]:
```suggestion
* View clients with managed configuration files
```

File: modules/reference/pages/configuration/config-overview.adoc [hunk 6]
```diff
@@ -10,15 +10,28 @@ image::configuration_overview.png[scaledwidth=80%]
...
+* View All Managed Configuration Files
```
Comment (Loquacity) [inline modules/reference/pages/configuration/config-overview.adoc:25]:
```suggestion
* View all managed configuration files
```

File: modules/reference/pages/configuration/config-overview.adoc [hunk 7]
```diff
@@ -10,15 +10,28 @@ image::configuration_overview.png[scaledwidth=80%]
...
+* View All Managed Configuration Channels
```
Comment (Loquacity) [inline modules/reference/pages/configuration/config-overview.adoc:26]:
```suggestion
* View all managed configuration channels
```

File: modules/reference/pages/configuration/config-overview.adoc [hunk 8]
```diff
@@ -10,15 +10,28 @@ image::configuration_overview.png[scaledwidth=80%]
...
+* Create a New Configuration Channel
```
Comment (Loquacity) [inline modules/reference/pages/configuration/config-overview.adoc:27]:
```suggestion
* Create a new configuration channel
```

File: modules/reference/pages/configuration/config-overview.adoc [hunk 9]
```diff
@@ -10,15 +10,28 @@ image::configuration_overview.png[scaledwidth=80%]
...
+* Enable Configuration Management on Systems
```
Comment (Loquacity) [inline modules/reference/pages/configuration/config-overview.adoc:28]:
```suggestion
* Enable configuration management on clients
```

**Comment by keichwa** [comment]
On 2020-05-14 01:54, Lana Brindley wrote:
> @Loquacity requested changes on this pull request.
> 
> so many initcaps!

Yes, this is copypasta from the web ui.  And yes, we can change it.

-- 
Karl Eichwalder

File: modules/reference/pages/configuration/config-channels.adoc [hunk 10]
```diff
@@ -1,25 +1,24 @@
 [[ref.webui.config.channels]]
-= Configuration Channels
+=  Channels
```
Comment (Loquacity) [inline modules/reference/pages/configuration/config-channels.adoc:2]:
```suggestion
= Channels
```

File: modules/client-configuration/pages/configuration-management.adoc [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+
+.Procedure: Creating a Configuration File, Directory, or Symbolic Link
+... Choose whether you want to create a text file, directory, or symbolic link in the [guimenu]``File Type`` section.
+... In the [path]``Filename/Path`` text box, set the absolute path to where the file should be deployed.
+... If you are creating a symbolic link, indicate the target file and path in the [guimenu]``Symbolic Link Target Filename/Path`` text box.
+... Enter the [guimenu]``User name`` and [guimenu]``Group name`` for the file in the [guimenu]``Ownership`` section, and the [guimenu]``File Permissions Mode``.
+... If the client has SELinux enabled, you can configure [guimenu]``SELinux contexts`` to enable the required file attributes (such as user, role, and file type).
+... If the configuration file includes a macro, enter the symbol that marks the beginning and end of the macro.
+... Then enter the configuration file content in the [guimenu]``File Contents`` field, using the script drop-down box to choose the appropriate scripting language.
+... Click the btn:[Create Configuration File] button to create the new file.
```
Comment (Loquacity) [inline modules/client-configuration/pages/configuration-management.adoc:87]: This whole procedure needs a re-write.

**Comment by keichwa** [comment]
On 2020-05-15 02:06, Lana Brindley wrote:
> +.Procedure: Creating a Configuration File, Directory, or Symbolic Link
> ...
> +... Click the btn:[Create Configuration File] button to create the new 
> file.
> 
> This whole procedure needs a re-write.

:+1: Agreed.  Must wait until next week ;)

-- 
Karl Eichwalder                    SUSE Software Solutions Germany GmbH
                                    Maxfeldstraße 5
                                    90409 Nürnberg, Germany
Geschäftsführer: Felix Imendörffer (HRB 36809, AG Nürnberg)

File: modules/client-configuration/pages/configuration-management.adoc (hunk 11 above)
Comment (keichwa) [reply modules/client-configuration/pages/configuration-management.adoc]: this is in now

File: .changelog [hunk 12]
```diff
@@ -5,6 +5,8 @@
 
 # Last packaged: 2020-05-19
 
+- Updated Configuration chapter in References
```
Comment (Loquacity) [inline .changelog:8]:
```suggestion
- Updated Configuration chapter in Reference Guide
```

File: modules/reference/pages/configuration/config-menu.adoc [hunk 13]
```diff
@@ -1,62 +1,18 @@
 [[ref.webui.config]]
 = Configuration Menu
 
-Only Configuration Administrators or {productname} Administrators see the menu:Main Menu[Configuration] pages.
+The [guimenu]``Configuration`` menu provides access to features for managing the configuration of {productname} clients.
```
Comment (Loquacity) [inline modules/reference/pages/configuration/config-menu.adoc:4]:
```suggestion
The [guimenu]``Configuration`` section provides access to features for managing the configuration of {productname} clients.
```

File: modules/client-configuration/pages/configuration-management.adoc [hunk 14]
```diff
@@ -0,0 +1,88 @@
+// From reference guide
```
Comment (Loquacity) [inline modules/client-configuration/pages/configuration-management.adoc:1]: I haven't reviewed this section.

**Pull request merged**